go/install/install_osx.go

```diff
@@ -417,13 +417,26 @@ func Install(context Context, binPath string, components []string, force bool, l
 	}
 
 	if libkb.IsIn(string(ComponentNameKBFS), components, false) {
-		err = InstallKBFS(context, binPath, force, log)
+		if !isKBFSCompatible(log) {
+			err = fmt.Errorf("Oops, the Keybase Filesystem isn't currently available on MacOS 10.12 (Sierra). We are working on a fix which should be available shortly.")
+		} else {
+			err = InstallKBFS(context, binPath, force, log)
+		}
 		componentResults = append(componentResults, componentResult(string(ComponentNameKBFS), err))
 	}
 
 	return newInstallResult(componentResults)
 }
 
+func isKBFSCompatible(log Log) bool {
+	osver, err := OSVersion()
+	if err != nil {
+		log.Warning("Error getting OS version: %s", err)
+		return false
+	}
+	return !(osver.Major == 10 && osver.Minor == 12)
```

Comment:
I think it's ok to target 10.12 exactly... Fuse has to be whitelisted for each OS version anyway so...